omnibus/files/private-chef-upgrades/001/037_elasticsearch_to_opensearch_migration.rb

```diff
@@ -0,0 +1,38 @@
+define_upgrade do
+    if Partybus.config.bootstrap_server
+  
+      es = Partybus.config.running_server["private_chef"]["opensearch"]
+      # Run these migrations only if elasticsearch has been enabled in the config
+      # We do not want to run this if solr is enabled
+  
+      if es["enable"]
+  
+        must_be_data_master
+
+        sleep 30
+  
+        log "All orgs are in the 503 mode..."
```

Comment:
How do we ensure that all orgs are in fact in 503 mode?

Reply:
Not sure exactly,as per knowledge. 
This migration will be executed when the customer upgrade from the older version.In the upgrade process,the very first step is to stop all the services it means that no service running while this process unless we start some serivces intentionally for migrations using `start_services(["nginx", "opscode-erchef", "opensearch"])` or `stop_services(["nginx", "opscode-erchef", "opensearch"])`.

I can remove this If it is misleading.